imports/blocks/add-schedule/index.js

```diff
@@ -102,23 +141,23 @@ class CartContainer extends Component {
     this.props.saveSchedule(id, {
       label: name,
       frequency: this.state.frequency,
-      start: this.state.start,
+      start: this.state.startDate,
```

Comment:
interesting. I wonder if this is causing the wrong schedule start date but I have heard about a few times?